Song/Koballoon.i7x

```diff
@@ -170,14 +170,26 @@ to say KoballoonHyperLoss: [requires Twisted Capacity]
+		say "     Soon their rhythm starts to change, becoming more erratic with each thrust, a sure sign they're reaching towards their climaxes. With your limited range of movement it's all you can do to brace yourself for the tide, as you know from experience that anything less would have you washed away. When they hit the point of no return, you can feel your own orgasm burning through you as you're pumped beyond full of their loads. For a brief moment it's almost as if you cease to exist, but they finally finish depositing their seed within you, and you sink back down from the pleasure high. In place of the ecstasy is instead a deep satisfaction from being so suffused with their essence, only slightly marred by the fact that they then pull out of you. You whine unconsciously at the void they leave within you, but when more hyper koballoons step into their place, you realize this is only the beginning.";
+		WaitLineBreak;
+		say "     Time seems to slip by easily, one moment flowing easily into the next. It's at least been hours, but it may have been days, and you have no way of knowing. After all, when you're almost constantly being plowed by one koballoon or another, there ceases to be a way to tell time, not that you even need one anymore. For the most part you're busy servicing them, with the only breaks truly coming when you've managed to work your way through all the available lizards, and instead must wait for others to return from outside. Hunger and thirst are no impediments to your constant worship, as whatever magic of theirs allowed you to be shaped into the inflated toy you are now seems to minimize that, and you're otherwise able to subsist on the very generous helpings of their own treats as you go.";
+		say "     Even in your sleep you still do your part for the gang, as many of them favor sleeping right alongside you with their members sheathed gently within. No matter the time or the way you're there to serve any need they have, and both the normal and hyper koballoons alike are always taking you up on the offer. Your elasticized body is perfectly suited to be pounded without end and never get tired, and you are able to constantly put it to the test with the assistance from your rubbery companions. Their constant efforts have also ensured that you stay pumped up for their enjoyment at all times, perhaps even more so than you used to be. At this point there isn't any other way you'd want to live than as the inflated sex toy you've become.";
 		decrease score by 25;
 		now humanity of Player is 0;
 		follow the turnpass rule;
-		end the story saying "A";
-		now battleground is "void";]
+		end the story saying "In the end you've become a live-in fuck-blimp for the enjoyment of the koballoons, and you couldn't be happier.";
```

Comment:
Might wanna add a new game ending for that bad end here?
See [How to manage Game Endings](https://gist.github.com/Stadler76/31e601cfd9a3238e65bfe7a0ba93e042#how-to-manage-game-endings).
If you're struggling with that, feel free to ping me on Discord.